tests/review.t

```diff
@@ -16,7 +16,7 @@ Setup. Let's pretend we have some outdated package versions installed:
 
 Also install library, which caused warning message:
 
-  $ pip install http://www.effbot.org/media/downloads/cElementTree-1.0.5-20051216.tar.gz >/dev/null 2>&1
+  $ pip install http://www.effbot.org/media/downloads/cElementTree-1.0.5-20051216.tar.gz >/dev/null 2>&1 || true
```

Comment:
Your commit message claims to "Ignore exit code of know bad pip command" (sic). But `pip install http://valid.domain.xyz/valid/path/ending/with.tar.gz` is not a "known bad pip command". To the contrary! This command is perfectly valid and it is in the tests for a reason. If it fails you can't just ignore that; you have to do something about it.